### PR TITLE
Add timeout parameter for url checker 

### PR DESCRIPTION
**Pull request by dplesca**
I took a stab at fixing #5.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,14 +38,15 @@ import (
 func main() {
     database, _ := sql.Open("mysql", "/")
 	mysql := db.NewMySQLChecker(database)
+    timeout := 5 * time.Second
     
     companies := health.NewCompositeChecker()
-    companies.AddChecker("Microsoft", url.NewChecker("https://www.microsoft.com/"))
-    companies.AddChecker("Oracle", url.NewChecker("https://www.oracle.com/"))
-    companies.AddChecker("Google", url.NewChecker("https://www.google.com/"))
+    companies.AddChecker("Microsoft", url.NewChecker("https://www.microsoft.com/", timeout))
```
Comment (dimiro1) [inline README.md]: I think we should not break the API. I prefer keep the url.NewChecker intact and create another url.NewCheckerWithTimeout and add this second parameter.

File: url/checker.go [hunk 2]
```diff
@@ -2,30 +2,34 @@ package url
...
 type Checker struct {
-	URL string
+	URL     string
+	Timeout time.Duration
 }
 
 // NewChecker returns a new url.Checker with the given URL
-func NewChecker(url string) Checker {
-	return Checker{URL: url}
+func NewChecker(url string, timeout time.Duration) Checker {
```
Comment (dimiro1) [inline url/checker.go]: See comment above.

Comment (dplesca) [inline url/checker.go]: What should the default value be?

Comment (dimiro1) [inline url/checker.go]: 5 seconds is a good number.

**Comment by dimiro1** [comment]
Looks good. Merging...

**Pull request merged**